magic_admin/resources/token.py

```diff
@@ -130,12 +132,21 @@ def validate(cls, did_token):
             did_token (base64.str): Base64 encoded string.
 
         Raises:
-            DIDTokenError: If DID token fails the validation.
+            DIDTokenInvalid: If DID token fails the validation.
+            DIDTokenExpired: If DID token has expired.
 
         Returns:
             None.
         """
         proof, claim = cls.decode(did_token)
+
+        if claim['ext'] is None:
+            raise DIDTokenInvalid(
+                message='Given DID token cannot be used at this time. Please '
+                'check the "ext" field and regenerate a new token with a suitable '
+                'value.',
```

Comment:
I think the first line could go away. The "at this time" makes me think of a temporal issue vs malformed.